fvm/environment/accounts_status.go

```diff
@@ -58,6 +71,23 @@ func (a *AccountStatus) ToBytes() []byte {
 // AccountStatusFromBytes constructs an AccountStatus from the given byte slice
 func AccountStatusFromBytes(inp []byte) (*AccountStatus, error) {
 	var as AccountStatus
+
+	if len(inp) == oldAccountStatusSize {
```

Comment:
This is the entirety of the compatibility code, to do account state migration on the fly.